mubench.reviewsite/templates/includes/detector_review_state.phtml

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @var array $exp_det_reviewstate contains information about reviewstate ordered by experiment->detector->NEEDS_REVIEW/NEEDS_CLARIFICATION/DISAGGREMENT
+ * @var Experiment $experiment the selected experiment
+ * @var Detector $detector the selected detector
+ */
+
+use MuBench\ReviewSite\Models\Detector;
+use MuBench\ReviewSite\Models\Experiment;
+use MuBench\ReviewSite\Models\ReviewState;
+
+if ($exp_det_reviewstate[$experiment->id][$detector->id][ReviewState::NEEDS_REVIEW]): ?>
```

Comment:
Can we somehow distinguish whether I can do another review here or whether all required reviews need to be done by somebody else? I don't habe a really good idea from the top of my head, but, as a reviewer, I would really like to see this.

Reply:
The only ideas I have right now are either color coding it or maybe showing the number of reviews you can still review yourself (if any) next to the icon.

Reply:
We could change the icon from red to grey if there's no reviews that I can do, yes. That's the best idea so far... I'm trying to think of other systems that have a similar kind of notification badges, but I can't really think of one that shows a badge for both me personally and all users of the system.